infra/build/functions/request_build.py

```diff
@@ -95,7 +94,7 @@ def request_build(event, context):
 
   with ndb.Client().context():
     credentials, cloud_project = google.auth.default()
-    build_steps = get_build_steps(project_name, cloud_project, BASE_PROJECT)
+    build_steps = get_build_steps(project_name, cloud_project)
```

Comment:
[Function `get_build_steps()`](https://github.com/google/oss-fuzz/blob/3bda7ff48186dc5b6522b218f40195931d97eb65/infra/build/functions/request_build.py#L68) expects a third parameter `base_images_project`.

I reckon this is why `presubmit` fails.